tests/testthat/test-proportional_hazards-survival.R

```diff
@@ -86,10 +86,13 @@ test_that("survival predictions - stratified", {
   f_fit <- fit(cox_spec,
                Surv(stop, event) ~ rx + size + number + strata(enum),
                data = bladder)
+  exp_f_fit <- coxph(Surv(stop, event) ~ rx + size + number + strata(enum),
+                     data = bladder, x = TRUE)
 
   new_data_3 <- bladder[1:3, ]
   f_pred <- predict(f_fit, new_data = new_data_3,
                     type = "survival", time = c(10, 20))
+  exp_f_pred <- pec::predictSurvProb(exp_f_fit, new_data_3, times = c(10, 20))
```

Comment:
Can yo `dput()` this or save it as an object? I'd like to get rid of this dependency eventually.